Project-Euler/Problem022.js

```diff
@@ -1,3 +1,5 @@
+import * as fs from 'fs'

```

Comment:
Why not just import `readFileSync` from fs? It seems to be the only function you need.